tests/middleware/tests.py

```diff
@@ -276,6 +276,20 @@ def test_no_etag_streaming_response(self):
         self.assertFalse(CommonMiddleware().process_response(req, res).has_header('ETag'))
 
     @override_settings(USE_ETAGS=True)
+    def test_no_etag_no_store_cache(self):
+        req = HttpRequest()
+        res = HttpResponse('content')
+        res['Cache-Control'] = 'no-store'
```

Comment:
To make this a better test, use multiple values and varying case. E.g. `'No-Cache, No-Store, Max-age=0'`.